scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -121,6 +123,11 @@ case class Args(
       "The Scala compiler version that was used to compile this project."
     )
     scalaVersion: String = scala.util.Properties.versionNumberString,
+    @Description(
+      "default Scala213" +
+        "Possibilities: Scala3, Scala213"
```

Comment:
doesn't metaconfig generate this out of the box?

Reply:
Ah, I don't know